### PR TITLE
[MMM-Spotify.js] Add FA Icons | Custom Icons by Device Type | Album Info | Modularize Code

### PR DESCRIPTION
**Pull request by humbertovirtudes**
Adds FA Icons
Adds Album Info
Re formats code, introduces modularity

 -  getIconContainer
 - getHTMLElementWithID
 - getDeviceContainer
 - getControlsContainer

Adds different icons based on device type

### CODE REVIEW

**Comment by eouia** [comment]
Thanks for your contribution.

**Comment by humbertovirtudes** [comment]
> Thanks for your contribution.

yw!

would it be ok If I also moved the timers to look like the below? (side by side with the progress bar)

current  /        progress bar     /    song length
    --:-- |===============| --:--

File: MMM-Spotify.js [hunk 1]
```diff
@@ -298,149 +310,189 @@ Module.register("MMM-Spotify", {
...
+
+  getControlsContainer() {
+    const control = this.getHTMLElementWithID('div', "SPOTIFY_CONTROL")
+
+    const shuffle = this.getHTMLElementWithID('div', "SPOTIFY_CONTROL_SHUFFLE")
+    shuffle.addEventListener("click", () => { this.clickShuffle() })
+    shuffle.className = "off"
+
+    shuffle.appendChild(this.getIconContainer('iconify', null, 'mdi:shuffle'))
+
```
Comment (humbertovirtudes) [inline MMM-Spotify.js:391]: nit: modularize this as well

**Comment by humbertovirtudes** [comment]
@eouia what happened?

**Comment by bugsounet** [comment]
that's ok, it's a website Spotify bug

I commit it :)

@eouia have gone programming module

thanks for your contribution

**Pull request merged**